README.md

```diff
@@ -18,3 +18,4 @@ This project is designed for one to practice the Git/Github workflow here at Lam
 ## ADD your name here
 
 - Steve Rogers
+- Cecil John Tantay
```

Comment:
Thanks for the collaboration invite and going through this practice project, CJ.